SPECS/cf-cli/cf-cli.spec

```diff
@@ -32,6 +32,7 @@ Source0:        https://github.com/cloudfoundry/cli/archive/refs/tags/v%{version
 #       - For the value of "--mtime" use the date "2021-04-26 00:00Z" to simplify future updates.
 Source1:        cli-%{version}-vendor.tar.gz
 Patch0:         CVE-2023-39325.patch
+Patch1:         CVE-2024-24786.patch
```

Comment:
We are not using `autopatch` in this spec, so we have to either add `autopatch` support or explicity patch